airflow/providers/google/cloud/secrets/secret_manager.py

```diff
@@ -101,9 +104,13 @@ def __init__(
                     "`connections_prefix`, `variables_prefix` and `sep` should "
                     f"follows that pattern {SECRET_ID_PATTERN}"
                 )
-        self.credentials, self.project_id = get_credentials_and_project_id(
-            keyfile_dict=gcp_keyfile_dict, key_path=gcp_key_path, scopes=gcp_scopes
-        )
+        try:
+            self.credentials, self.project_id = get_credentials_and_project_id(
+                keyfile_dict=gcp_keyfile_dict, key_path=gcp_key_path, scopes=gcp_scopes
+            )
+        except Exception as e:
+            log.exception(f'{e}')
```

Comment:
`Logger.exception()` already logs the exception, so the `f'{e}'` message is adding nothing useful. You should provide a readable string explaining why the exception happens instead.